masonry/src/widgets/flex.rs

```diff
@@ -19,7 +19,7 @@ use crate::properties::{Background, BorderColor, BorderWidth, CornerRadius, Padd
 use crate::theme::DEFAULT_GAP;
 use crate::util::{debug_panic, fill, include_screenshot, stroke};
 
-/// A container with either horizontal or vertical layout.
+/// A flexible container with either horizontal or vertical layout.
```

Comment:
Might be worth calling out here that if you don't need features blah blah blah that you can use `Stack` instead.